.gitignore

```diff
@@ -28,3 +28,4 @@ go.work
 
 # Temporary Kustomize file
 config/bpfman-deployment/kustomization.yaml
+config/bpfman-operator-deployment/kustomization.yaml
```

Comment:
No, you don't want this. This file is showing up as changed as a result of the `$(KUSTOMIZE) edit set image`, but that file is still under source control. I was thinking about replacing the `$(KUSTOMIZE) edit set image` with the `sed` command as well for that reason, but didn't because it is working as designed. The other file was just broken.

Reply:
ok will remove it